README.md

```diff
@@ -1,32 +1,51 @@
 # Pylint extension for Visual Studio Code
 
-A Visual Studio Code extension with support for the `pylint` linter. The extension ships with `pylint=3.0.1`.
+A Visual Studio Code extension with support for the Pylint linter. This extension ships with `pylint=3.0.1`.
 
-Note:
+This extension supports all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of the `Python` language (i.e., Python >= 3.8).
 
```

Comment:
I think it would be nice to provide a link for more info like we do for Mypy.
```suggestion

For more information on Pylint, see https://pylint.readthedocs.io/
```